src/box/xrow.h

```diff
@@ -228,12 +236,33 @@ request_str(const struct request *request);
  * @param[out] request DML request to decode to.
  * @param key_map a bit map of keys that are required by the caller,
  *        @sa request_key_map().
+ * @param accept_space_name space name is accepted instead of space identifier.
  * @retval 0 on success
  * @retval -1 on error
  */
 int
+xrow_decode_dml_internal(struct xrow_header *xrow, struct request *request,
+			 uint64_t key_map, bool accept_space_name);
+
+/**
+ * Decode DML from system request (recovery or replication).
+ */
+static inline int
 xrow_decode_dml(struct xrow_header *xrow, struct request *request,
-		uint64_t key_map);
+		uint64_t key_map)
+{
+	return xrow_decode_dml_internal(xrow, request, key_map, false);
+}
+
+/**
+ * Decode DML from client request (Lua, C or IPROTO).
```

Comment:
Nit: AFAIU this function is only used by Lua clients.

Reply:
And what function do IPROTO clients use?

Reply:
Actually, I guess this function is only used by IPROTO clients (perhaps you meant the same). Renamed the function to `xrow_decode_dml_iproto` and changed the comment accordingly.